rules/macos/credential_access_mitm_localhost_webproxy.toml

```diff
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2021/01/05"
+maturity = "production"
+updated_date = "2021/01/05"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the use of the networksetup built-in command to setup the webproxy settings. This may indicate an attempt to
+hijack Web Browsers traffic for credentials access via Sniffing or redirection.
```

Comment:
```suggestion
Identifies the use of the built-in networksetup command to configure webproxy settings. This may indicate an attempt to
hijack web browser traffic for credentials access via traffic sniffing or redirection.
```